mautrix_telegram/__main__.py

```diff
@@ -112,7 +115,10 @@ def prepare_bridge(self) -> None:
 
         if self.config['metrics.enabled']:
             self.as_connection_metric_task = self.loop.create_task(self._loop_check_as_connection_pool())
-        
+
+        if self.config['telegram.liveness_timeout'] and self.config['telegram.liveness_timeout'] >= 1:
+            self.as_bridge_liveness_task = self.loop.create_task(self._loop_check_bridge_liveness())
```

Comment:
Not sure how this project works, but in Python generally it's considered good practice to define all your class instance variables before setting or checking against them at some point later. I would suggest adding a `as_bridge_liveness_task = None` to the class definition to indicate that this instance variable is going to possibly be defined at some point.

Reply:
Set to None in d4b5e0b.